crates/sel4-capdl-initializer/core/src/lib.rs

```diff
@@ -359,6 +359,28 @@ impl<'a, N: ObjectName, D: Content, M: GetEmbeddedFrame, B: BorrowMut<[PerObject
                                 }
                             }
                         }
+                        #[sel4_cfg(ARCH_X86_64)]
+                        Object::IrqMsi(obj) => {
+                            init_thread::slot::IRQ_CONTROL.cap().irq_control_get_msi(
+                                obj.extra.pci_bus,
+                                obj.extra.pci_dev,
+                                obj.extra.pci_func,
+                                obj.extra.handle,
+                                *irq,
+                                &cslot_to_relative_cptr(slot),
+                            )?;
+                        
```

Comment:
Is a `}` missing here?

Reply:
Ugh, sorry I was lazy and didn't check that it compiled.

Reply:
(after changing the cfg change)

Reply:
It compiles for me locally now.